### DISCUSSION TITLE
How do you create a new container sandbox?

### DISCUSSION THREAD

**Discussion by jd-fillmore** [codesandbox/codesandbox-client#6873] [General]
I've scoured the docs, Google, and I can't find anything on how to create a new container sandbox.

The only area that mentions containers is [here](https://codesandbox.io/docs/environment), and it only explains what it is and how to tell if you're in one.

**Comment by archiewood** [comment]
Came here to ask this question!

**Comment by alexnm** [comment]
If you are looking for the new container experience (cloud sandboxes), when creating a sandbox, you can see a `Cloud Beta` badge next to the templates that will create a new cloud sandbox.

<img width="733" alt="image" src="https://user-images.githubusercontent.com/9945366/200006571-837c232b-339e-4358-a8d8-9b02d1d8f286.png">
